rootfs-configs.yaml

```diff
@@ -32,3 +32,58 @@ rootfs_configs:
       - partx
       - find
     script: ""
+  buster-v4l2:
```

Comment:
One patch for this one, to add v4l2.  Also please include a blank line between rootfs configs for readability.